packages/features/bookings/components/AvailableTimes.tsx

```diff
@@ -95,7 +95,7 @@ export const AvailableTimes = ({
 };
 
 export const AvailableTimesSkeleton = () => (
-  <div className="mt-8 flex w-[20%] flex-col only:w-full">
+  <div className="flex w-[20%] flex-col only:w-full">
```

Comment:
Remove mt-8 as the skeleton was starting way too far down the page

Reply:
only question is, if we should increase the height for column view skeletons

I think the height is slightly different right now